source/api_documentation/network_integration/advertiser_campaigns/_get_advertiser_campaign_unarchive.rst

```diff
@@ -0,0 +1,15 @@
+.. container:: endpoint-long-description
```

Comment:
Yes, I agree with you. I just replicated the examples above with go_live, where there's also a GET and a POST example. I think we should remove the GET example for the archive action. Would be ideal to remove the GET for go_live as well, but that's not possible without a version bump.
@mikeweaver, @spencerpbartz Do we also want to update the API endpoint to not allow GET request? This would be safe for the archive action since that wasn't exposed in the API doc till now.